.github/workflows/check-missing-files.yml

```diff
@@ -0,0 +1,64 @@
+name: Checking missing files
+
+on:
+  schedule:
+    - cron: '0 0 * * *'
+  workflow_dispatch:
+    inputs:
+      dryRun:
+        description: 'Indicates whether to trigger changes to a product. By default, `dryRun` is set to `true`, meaning the action will perform a check without applying changes. When set to `false`, the action will both check for and add any missing files to the products.'
+        default: 'true'
+      workingOrgs:
+        description: 'Define organizations must be checked, example: axonivy,axonivy-market'
+        default: 'axonivy,axonivy-market'
+  push:
+    branches: [ "master" ]
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+
+    steps:
+
+      - name: Checkout Code
+        uses: actions/checkout@v4
+
+      - name: Setup Java JDK
+        uses: actions/setup-java@v4
+        with:
+          java-version: 17
+          distribution: temurin
+
+      - name: Setup Maven
+        uses: stCarolas/setup-maven@v5
+
+      - name: Create a file with TOKEN value
+        run: |
+          echo -n "${{ secrets.TOKEN }}" > token_file
+          echo "File created at: $(pwd)/token_file"
+
+      - name: Get the file path
+        id: get-file-path
+        run: echo "::set-output name=filepath::$(pwd)/token_file"
+
+      - name: Set default values for dryRun and workingOrgs
+        id: set-defaults
+        run: |
+          echo "dryRun=${{ github.event.inputs.dryRun || 'true' }}" >> $GITHUB_ENV
+          echo "workingOrgs=${{ github.event.inputs.workingOrgs || 'axonivy,axonivy-market' }}" >> $GITHUB_ENV        
+
+      - name: Build with Maven
+        working-directory: ./github-repo-manager
+        run: |
+          mvn -f pom_check_missing_file.xml -B clean compile exec:java \
+            -DDRY_RUN="${{ env.dryRun }}" \
+            -DGITHUB.TOKEN.FILE="${{ steps.get-file-path.outputs.filepath }}" \
```

Comment:
this token passing seems dangerous and adventurous, I'd rather just set an environment variable in the pipeline .. and read it from the environment in java.
the pipeline will become easier and security will be increased.

Reply:
Thanks @ivy-rew , I will close the PR